modules/light-clients/08-wasm/types/vm.go

```diff
@@ -35,7 +35,7 @@ func initContract(ctx sdk.Context, clientStore storetypes.KVStore, codeHash []by
 		Funds:  nil,
 	}
 
-	ctx.GasMeter().ConsumeGas(VMGasRegister.NewContractInstanceCosts(len(msg)), "Loading CosmWasm module: instantiate")
+	ctx.GasMeter().ConsumeGas(VMGasRegister.NewContractInstanceCosts(false, len(msg)), "Loading CosmWasm module: instantiate")
```

Comment:
I think this should be `true` considering we `Pin`? wasmd does conditionally pass this based on `IsPinnedCode` but we do it unconditionally.

Reply:
we do save some gas from it apparently:

```
       dataCosts := storetypes.Gas(msgLen) * g.c.ContractMessageDataCost
	if pinned {
		return dataCosts
	}
	return g.c.InstanceCost + dataCosts
```

would be nice to ask Simon just for the verification.

Reply:
I think you're right, it probably should be true across the board. This will result in different gas costs. E.g. `dataCosts` instead of `g.c.InstanceCost + dataCosts` (which I believe is actually correct)

Will update this!